Add vision and audience to homepage (#56)
thegooddocsproject/thegooddocsproject.github.io: index.md

_“Best practice templates and writing instructions for documenting open source software.”_

### What?

Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. 

> We are creating processes, templates, and guides covering the numerous aspects of writing good documentation; and we are providing justifications by describing our logic and referencing research where available.

Reviewer: ```suggestion
We are creating processes, templates, and guides covering the numerous aspects of writing good documentation; we are providing justifications by describing our logic and referencing research where available.
```

Author: Thanks @Loquacity. That long sentence bothered me. I changed it into two sentences - is it any better now? I'm also bothered by the case.
Also curious to see what happens on GitHub to 'outdated' suggestions....

### Why?

We want to help open source projects be more effective and more impactful by helping them lift the quality of their documentation. 

Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of documentation using a suite of templates.

### Who?

We are writing for people who have an open source project and want to improve their docs.

Our target readers are software developers and early-stage technical writers, who can read and write English, and are motivated to learn by example.

### Download

* The latest development version of templates are available from the [github main branch](https://github.com/thegooddocsproject/templates).
* The [0.1 alpha straw-man release of templates](https://github.com/thegooddocsproject/templates/releases/tag/v0.1) was released at the 2019 Write The Docs - Australia conference.

### 2020 Initiatives
* Templates
* Information architecture guide
* Cross-domain management of glossaries 

> * How-to for style guides

Reviewer: ```suggestion
* How-to for style guides
* Base template (template for templates)
```

_“Best practice templates and writing instructions for documenting open source software.”_

### What?

> Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. 

Reviewer 2: ```suggestion
Every project deserves a minimum level of _Good Docs_, but it can be really hard to know where to start or what to write. 
```

> We are creating processes, templates, and guides covering the numerous aspects of writing good documentation. We describe our logic, referencing research where available.

Reviewer 2: ```suggestion
We are creating processes, templates, and guides covering the numerous aspects of writing good documentation. We describe our reasoning, referencing research where available, to help writers decide when they should deviate from suggestions.
```

> ### Why?

Reviewer 2: For a vision, I thinks this should be presented in the Why/How/What/(Who) format, in line with Simon Sinek's reasoning:
https://www.ted.com/talks/simon_sinek_how_great_leaders_inspire_action?language=en

Author: Nice Ted talk. I'll see what I can do.

> We want to help open source projects be more effective and more impactful by helping them lift the quality of their documentation. 

Reviewer 2: ```suggestion
We want to help open source projects be more effective and more impactful by helping them lift the quality of their documentation. And in the process, we expect to lift the quality of technical writing for all software.
```

Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of documentation using a suite of templates.

### Who?

> We are writing for people who have an open source project and want to improve their docs.

Reviewer 2: ```suggestion
We are writing for people who have an open source project and want to improve their docs.
```
Our key target audiences are:
* **Software developers** wanting to follow guides and templates as they create documentation for their project.
* **Technical writers** setting up a documentation framework, who wish to make use of established templates and processes, customizing where appropriate to align with the project's goals.
* **Project owners**  who need balance project priorities, and want to understand the return-on-effort associated with different documentation strategies.

Author: Okay, I've incorporated this.

> We are writing for people who have an open source project and want to improve their docs.

Reviewer 2: ```suggestion
```
I think we can drop this line now, as we have it covered below and above.

Author: Done.

> Our target readers are software developers and early-stage technical writers, who can read and write English, and are motivated to learn by example.

Reviewer 2: ```suggestion
```
You can drop this line if you accept line above.

Author: Done.

### Download

* The latest development version of templates are available from the [github main branch](https://github.com/thegooddocsproject/templates).
* The [0.1 alpha straw-man release of templates](https://github.com/thegooddocsproject/templates/releases/tag/v0.1) was released at the 2019 Write The Docs - Australia conference.

### 2020 Initiatives

> * Templates

Reviewer 2: ```suggestion
The following initiatives are being worked on in 2020:
* Templates
```

_“Best practice templates and writing instructions for documenting open source software.”_

### Why?

> We want to help open source projects be more effective and more impactful by helping them lift the quality of their documentation. In the process, we expect to lift the quality of technical writing for all software.

Reviewer 3: what's the difference between effective and impactful here? :) Probably better to just leave as "more effective". 

Also I think better to take out the "In the process ..." sentence. We may expect this but it's kind of a lofty goal. Let's underpromise and overdeliver! :)

Author: Hmm, Cameron added that sentence in. He has lofty goals :-)  I'll remove it and let you two duke it out :-)

### How?

We are creating processes, templates, and guides covering the numerous aspects of writing good documentation. We describe our reasoning, referencing research where available, to help writers decide when they should deviate from suggestions.

### What?

Every project deserves a minimum level of _Good Docs_, but it can be really hard to know where to start or what to write. Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of documentation using a suite of templates.

> 

Reviewer 4: I love all your changes! I'm curious if we have defined our MVD yet. If so, perhaps a link would help here.

Author: Oh man, yeah. I was curious about this sentence too. I don't think we've used this phrase anywhere else, and I don't think we've defined the MVD, but I could be wrong. 
Hmm, how about "Our open source suite of templates will help you create a baseline set of documentation."
